draft-ietf-regext-rdap-geofeed.md

```diff
@@ -66,36 +79,42 @@ to as a Geofeed link object) in its "links" array, with the following REQUIRED J
 
 * "value" -- The "value" JSON value is the context URI ([@!RFC9083, section 4.2]).
 * "rel" -- The "rel" JSON value is the link relation type and set to the "geo" string. The "geo" link relation type is
-new and will be registered in the IANA Link Relations Registry (see the "Link Relations Registry" section).
+new and will be registered in the IANA Link Relations Registry (see (#link_relations_registry)).
 * "href" -- The "href" JSON value is the target URI and set to the HTTPS URL of the geofeed file for an IP network.
 
 Per the definition of a web link ([@!RFC8288]), a Geofeed link object may have additional JSON members.
 Specifically:
 
 * "type" -- The "type" JSON value is the media type for the target URI. Given that the geofeed data is mostly intended
 for use by automated/scripted processes, it is RECOMMENDED that server operators set a media type in Geofeed link
-objects. See the "Media Type for a Geofeed Link" section for acceptable "type" values.
+objects. See (#media_type_for_a_geofeed_link) for acceptable "type" values.
 * "hreflang" -- The "hreflang" JSON value is an attribute for the target URI and could be used to indicate the languages
 the geofeed data is available in. It is OPTIONAL.
 
 There MAY be zero or more Geofeed link objects in the "links" array of an IP Network object. In other words, the Geofeed
 link objects are OPTIONAL.
 
-## Media Type for a Geofeed Link
+## Media Type for a Geofeed Link {#media_type_for_a_geofeed_link}
 
 [@?I-D.ymbk-opsawg-9092-update] requires a geofeed file to be a UTF-8 CSV file, with a series of "#" comments at the end
 for the optional RPKI (Resource Public Key Infrastructure) signature. At first glance, the "text/csv" media type
-([@!RFC7111, section 5.1]) seems like a good candidate to represent a geofeed file, but it does not support the "#"
-comments needed to include the RPKI signature.
+([@?I-D.shafranovich-rfc4180-bis, section 4]) seems like a good candidate to represent a geofeed file since it now
+supports the "#" comments needed for including the RPKI signature.
+
+However, although the CSV geofeed data could be directly viewed by a user but to make it truly usable, that data, along
```

Comment:
"However, although the CSV geofeed data could be directly viewed by a user, the most common use case will involve it being processed by some sort application first, in order to facilitate subsequent address lookup operations.  Therefore, using ...".